api/alembic/versions/6f6effb6b004_add_complete_flag_to_prediction_model.py

```diff
@@ -0,0 +1,34 @@
+"""Add complete flag to prediction model
+
+Revision ID: 6f6effb6b004
+Revises: 8bca5e25546e
+Create Date: 2020-08-17 17:27:45.878364
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '6f6effb6b004'
+down_revision = '8bca5e25546e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add "complete" column.
+    op.add_column('prediction_model_run_timestamps', sa.Column(
+        'complete', sa.Boolean(), nullable=True))
+    # Mark all rows as completed. (They might not be, but let's just assume....)
+    op.execute('update prediction_model_run_timestamps set complete = True')
```

Comment:
Just read the article about this.
It's kind of annoying that you have to do this :(